src/token.sw

```diff
@@ -25,3 +27,66 @@ pub fn force_transfer(amount: u64, asset_id: ContractId, contract_id: ContractId
         tr r3 r1 r2;
     }
 }
+
+/// Transfer `amount` coins of type `asset_id` to address `recipient`.
+pub fn transfer_to_output(amount: u64, asset_id: ContractId, recipient: Address) {
+    // note: if tx format changes, the magic number "48" must be changed !
+    // TransactionScript outputsCount has a 48 byte(6 words * 8 bytes) offset
+    // Transaction Script: https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md#transactionscript
+    // Output types: https://github.com/FuelLabs/fuel-specs/blob/master/specs/protocol/tx_format.md#output
+    const OUTPUT_LENGTH_LOCATION = 48;
```

Comment:
I don't think 48 is right.  If I'm looking at the right table, there's `gasPrice`, `gasLimit`, `bytePrice` are 8 byte words, then `maturity` is a 4 byte word, then `scriptLength` and `scriptDataLength` are 2 byte words, and then `inputsCount` and `outputsCount` are bytes.

8+8+8+4+2+2+1 is 33.

Reply:
Everything is word aligned, so things less than a word would be aligned to a word. So it would be 8+8+8+8+8+8+8 = 56, since an extra field was added recently. Used to be 48.